src/Advection/cell_advection_timescale.jl

```diff
@@ -16,17 +16,17 @@ function cell_advection_timescale(grid, velocities)
     return minimum(τ)
 end
 
-@inline _inverse_timescale(i, j, k, Δ, U, topo) = @inbounds abs(U[i, j, k]) / Δ
-@inline _inverse_timescale(i, j, k, Δ, U, topo::Flat) = 0
+@inline _inverse_timescale(i, j, k, Δ⁻¹, U, topo) = @inbounds abs(U[i, j, k]) * Δ⁻¹
+@inline _inverse_timescale(i, j, k, Δ⁻¹, U, topo::Flat) = 0
 
 @inline function cell_advection_timescaleᶜᶜᶜ(i, j, k, grid::AbstractGrid{FT, TX, TY, TZ}, u, v, w) where {FT, TX, TY, TZ}
-    Δx = Δxᶠᶜᶜ(i, j, k, grid)
-    Δy = Δyᶜᶠᶜ(i, j, k, grid)
-    Δz = Δzᶜᶜᶠ(i, j, k, grid)
+    Δx⁻¹ = Δxᶠᶜᶜ⁻¹(i, j, k, grid)
```

Comment:
```suggestion
    Δx⁻¹ = Δx⁻¹ᶠᶜᶜ(i, j, k, grid)
```

because its like `Δx⁻¹` (at this location)